docs/reference/modules/discovery/quorums.asciidoc

```diff
@@ -33,10 +33,17 @@ This means that if there are three or four master-eligible nodes, the cluster
 can tolerate one of them being unavailable. If there are two or fewer
 master-eligible nodes, they must all remain available.
 
-After a node has joined or left the cluster the elected master must issue a
-cluster-state update that adjusts the voting configuration to match, and this
-can take a short time to complete. It is important to wait for this adjustment
-to complete before removing more nodes from the cluster.
+If you stop half or more of the nodes in the voting configuration at the same
```

Comment:
nit A similar comment is at the top of https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-discovery-voting.html#modules-discovery-voting if you'd like to also cite that.

Reply:
Aha thanks Iraklis, refactored in 8b4cd220584.